configs/small-scale-node-density-cni.yml

```diff
@@ -0,0 +1,59 @@
+# This is a template file
+tests :
```

Comment:
I hit a small issue with the tiny url timing out when trying to run the below, increasing the timeout I was able to successfully run the command

` orion cmd --config examples/small-scale-cluster-density.yaml`

    `shortener = pyshorteners.Shortener(timeout=10)`